tests/unit/parsec/test_fileparse.py

```diff
@@ -554,3 +555,43 @@ def test_unclosed_multiline():
         assert exc.reason == 'Invalid line'
         assert 'echo hello world' in exc.line
         assert 'Did you forget to close [scheduling][graph]R1?' in str(exc)
+
+
+@pytest.mark.parametrize(
+    'expect, native_tvars, plugin_result, log',
+    [
+        pytest.param(
+            {'FOO': 123},
+            {'FOO': 123},
+            {
+                'templating_detected': None,
+                'template_variables': {'FOO': 123}
```

Comment:
Should make sure this value is ignored
```suggestion
                'template_variables': {'FOO': 122}
```